pkg/apis/core/helper/helpers.go

```diff
@@ -483,3 +484,17 @@ func IsMultiZonalSeed(seed *core.Seed) bool {
 	}
 	return seed.Spec.HighAvailability != nil && seed.Spec.HighAvailability.FailureTolerance.Type == core.FailureToleranceTypeZone
 }
+
+// ValidateMultiZoneSeedLabelValue validates the value of label `v1beta1constants.LabelSeedMultiZonal`.
+func ValidateMultiZoneSeedLabelValue(val string, fldPath *field.Path) field.ErrorList {
```

Comment:
```suggestion
func ValidateBooleanValue(val string, fldPath *field.Path) field.ErrorList {
```
You could create a generic function, because it is reused for checking seeds later on.

Reply:
Yes this can be done but this is so common that i would like to then move this to `pkg/apis/helper` package which is a set of common helpers across all apis. Will that be fine?

Reply:
> Also if this is going to be generic then i will have to return the error. Callers can determine if they wish to ignore the error or handle it explicitly.

Then there is no point of having this function anymore because people can just use `strconv.ParseBool`.
The benefit of this function is solely that it takes over the field handling part.

Reply:
I have now made the change. Will push with other changes.

Reply:
Changes are pushed.

Reply:
After the changes were made by @unmarshall and another discussion, we agreed to revert to `ValidateMultiZoneSeedLabelValue` again since there is an empty value case `len(val) == 0` that is very specific to the MultiZoneSeedLabel case. Apologies for the back and forth.

Reply:
change has been reverted now